czkawka_core/src/similar_images.rs

```diff
@@ -613,3 +667,103 @@ fn get_string_from_similarity(similarity: &Similarity) -> &str {
         Similarity::None => panic!(),
     }
 }
+
+fn save_hashes_to_file(hashmap: &HashMap<String, FileEntry>, text_messages: &mut Messages) {
+    if let Some(proj_dirs) = ProjectDirs::from("pl", "Qarmin", "Czkawka") {
+        // Lin: /home/alice/.config/barapp
```

Comment:
What is `barapp`? The `~/.config/barapp` doesn't exist, even if it looks like the cache works.

Reply:
Only this config looks wrong, the rest of the code and the cache work! :+1:

Reply:
This comment was intended to show only where this function want to save data in general(comment was copied from `directories-next` crate).
Proper locations I put to instruction.
I have updated this comments in https://github.com/qarmin/czkawka/issues/196